app/models/mixins/has_monitoring_manager_mixin.rb

```diff
@@ -1,21 +1,28 @@
 module HasMonitoringManagerMixin
   extend ActiveSupport::Concern
 
-  private
+  def ensure_monitoring_manager_with_params
```

Comment:
I liked the old name better. also why not keep private?

Reply:
1) we can keep the old name
2) cant be private since we access this through `Endpoint`

Reply:
I think ensure_monitoring_manager_with_params should be private and endpoint_created/endpoint_destroyed should be public, right?

Reply:
ah, sure.